app/controllers/v0/profile/service_histories_controller.rb

```diff
@@ -49,6 +50,14 @@ def raise_error!
           source: self.class.to_s
         )
       end
+
+      def report_results(response)
+        if response.present?
+          StatsD.increment("#{EMIS::Service::STATSD_KEY_PREFIX}.service_history.present")
```

Comment:
Just to clarify my comment, the two in this file would have the same metric name, with differing tag values.